docs/build/guides/transactions/simulateTransaction-Deep-Dive.mdx

```diff
@@ -12,15 +12,9 @@ While calling the method on the rpc server is not the ONLY way to simulate a tra
 
 Here we will look at the objects involved and their definitions.
 
-## RPC Servers
+## RPC Services
 
-Stellar Development Foundation provides a testnet RPC service at `http://soroban-testnet.stellar.org`. For public network providers please refer to the [Ecosystem RPC Providers] list. At the time of writing this documentation there are two public and free RPC providers, and many more you can subscribe to.
-
-### Public Network Endpoints:
-
-[Liquify](https://www.liquify.com): https://stellar-mainnet.liquify.com/api=41EEWAH79Y5OCGI7/mainnet
-
-[Gateway.fm](https://www.gateway.fm): https://soroban-rpc.mainnet.stellar.gateway.fm
+Stellar Development Foundation provides a testnet RPC service at `http://soroban-testnet.stellar.org`. For public network providers please refer to the [Ecosystem RPC Providers](docs/data/rpc/rpc-providers.mdx) list.
```

Comment:
```suggestion
Stellar Development Foundation provides a testnet RPC service at `http://soroban-testnet.stellar.org`. For public network providers please refer to the [Ecosystem RPC Providers](../../../data/rpc/rpc-providers.mdx) list.
```

Starting the link with just `docs/whatever` does _usually_ work but it can cause some build failures in some cases. Let's change this to a relative filepath.

Reply:
Sure thing